dags/openshift_nightlies/config/install/hypershift/sdn.json

```diff
@@ -7,12 +7,12 @@
     "managed_channel_group": "nightly",
     "managed_ocp_version": "latest",    
     "openshift_worker_count": 9,
-    "openshift_network_type": "OpenShiftSDN",
```

Comment:
this install config is meant for sdn type only, so instead we remove this dag.

Reply:
Agreed, removed